moped-editor/src/views/projects/projectView/ProjectNameComplex.js

```diff
@@ -0,0 +1,307 @@
+            <Typography
+              color="textSecondary"
+              variant="h2"
+              sx={{ display: "inline" }}
+            >
+              #{props.projectId}
+            </Typography>
+          </Box>
+
+          {/* Jogging this up 5 a bit to make it visually centered along the horizontal 
+          midline of the project name. */}
+          <Box sx={{ display: "inline", position: "relative", top: "-5px" }}>
+            <ProjectStatusBadge
+              phaseKey={props.currentPhase?.phase_key}
+              phaseName={props.currentPhase?.phase_name}
+            />
+          </Box>
+        </Grid>
+
+        {/* The secondary name field is shown conditionally, only if it is defined and not the 
+        empty string.  This prevents it from taking up height if it's not displaying anything. */}
+        {props.projectData.project_name_secondary &&
+        props.projectData.project_name_secondary.length > 0 ? (
+          <Grid item xs={12}>
+            <Box sx={{ display: "inline", cursor: "pointer" }}>
+              <Typography
+                color="textSecondary"
+                variant="h2"
+                sx={{ display: "inline", fontSize: "1.2rem" }}
+                onClick={() => props.setIsEditing(true)}
+              >
+                {props.projectData.project_name_secondary}
+              </Typography>
+            </Box>
+          </Grid>
+        ) : null}
+      </Grid>
+    </>
+  );
+
+  const editModeJSX = (
+    <>
+      {/* This grid contains the project name and secondary name fields, the accept/cancel icons 
+      and the badge.  Multiple widths are set for each element so that they reflow responsively 
+      as the viewport becomes more narrow. */}
+      <Grid container>
+        {/* Primary project name field */}
+        <Grid item xs={12} sm={6} sx={{ paddingRight: "30px" }}>
+          <TextField
+            variant="standard"
+            fullWidth
+            id="project_name"
+            label={"Project Name"}
+            type="text"
+            value={projectName}
+            error={primaryTitleError}
+            placeholder={
+              primaryTitleError ? "Title cannot be blank" : "Enter project name"
+            }
+            multiline={false}
+            rows={1}
+            onChange={handleProjectNameChange}
+            InputLabelProps={{
+              shrink: true,
+            }}
+            InputProps={{
+              classes: {
+                input: classes.projectNameEditField,
+              },
+            }}
+          />
+        </Grid>
+
+        {/* Secondary project name field */}
+        <Grid item xs={12} sm={3} sx={{ paddingRight: "30px" }}>
+          <TextField
+            variant="standard"
+            fullWidth
+            id="secondary_name"
+            label={"Secondary Name"}
+            type="text"
+            value={secondaryName}
+            placeholder={"Project Byline"}
```

Comment:
I'm not sold on the "Project Byline" placeholder, I think it would confuse people in thinking it should be the author of the project

Reply:
I think this is a great observation, and I changed the `placeholder` value to be "Secondary Name"